java/feature-layer-definition-expression/README.md

```diff
@@ -1,18 +1,34 @@
-# Feature Layer Definition Expression
-Limit which features to display on the map.
+# Feature layer definition expression
 
-![Feature Layer Definition Expression App](feature-layer-definition-expression.png)
+Limit the features displayed on a map with a definition expression.
+
+![Image of feature layer definition expression](feature-layer-definition-expression.png)
+
+## Use case
+
+Set a definition expression to filter out the features to be displayed. You might filter a dataset of tree quality selecting for only those trees which require maintenance or are damaged.
 
 ## How to use the sample
-Use the buttons in the bottom toolbar to apply or reset definition expression.
+
+Press the 'Apply Expression' button to limit the features requested from the feature layer to those specified by the SQL query definition expression. Click the 'Reset' button to remove the definition expression on the feature layer, which returns all the records.
 
 ## How it works
 
-Set the definition expression property on a feature layer using an SQL where clause to limit which features are displayed on the map.
+1. Create a service feature table from a URL.
```

Comment:
Both this and the following step could do with a code instruction I think

Reply:
I've added it, but just appending "with `instruction` always feels a bit off to me.

Reply:
I know what you mean, though it's in keeping with the majority of styles for other readmes, and draws the users eye to how to do it. For me this section stood out because of a lack of code instruction, so I think it's good to be consistent across the board. Another sign of our changing readme process over the last year!